inventory/vagrant/group_vars/webserver/drupal.yml

```diff
@@ -8,13 +8,15 @@ drupal_composer_dependencies:
   - "drupal/devel:^1.0@beta"
   - "drupal/rdfui:1.x-dev"
   - "drupal/restui:^1.13"
-  - "drupal/media_entity:^1.6"
-  - "drupal/media_entity_image:^1.2"
   - "drupal/search_api_solr:1.0.0"
   - "drupal/facets:1.x-dev"
   - "islandora/carapace:dev-8.x-1.x"
-  - "islandora/islandora_image:dev-8.x-1.x"
-drupal_composer_project_package: "islandora/drupal-project:8.4"
+  - "drupal/jwt:1.0.0-alpha6"
+# Comment this back in once code gets merged and is available on packagist
+#  - "islandora/islandora:dev-8.x-1.x"
+#  - "islandora/openseadragon:dev-8.x-1.x"
+#  - "islandora/islandora_image:dev-8.x-1.x"
```

Comment:
@dannylamb I think we are up-to-date in packagist. These can be uncommented now.